scribble-lib/scribble/example.rkt

```diff
@@ -94,11 +94,10 @@
                       (attribute no-result-kw)
                       (attribute no-form-kw))))
          (with-syntax ([srcloced-form srcloced-form]
-                       [title (or (attribute title)
-                                  (cond
-                                   [(= 1 (length (syntax->list #'(form ...))))
-                                    #'example-title]
-                                   [else #'examples-title]))])
+                       [title (cond
+                                [(attribute title) #t]
+                                [(= 1 (length (syntax->list #'(form ...)))) #'example-title]
+                                [else #'examples-title])])
```

Comment:
This change is buggy. If `(attribute title)` returns a non-`#f` and non-`#t` value, this change coerces it to `#t`.